### ISSUE TITLE
crypto-policies pinned to avoid Lua script

### ISSUE THREAD

**Issue by bgilbert** [coreos/fedora-coreos-tracker#540]
crypto-policies &ge; 20200527-3.gitb234a47 has a Lua script for FIPS mode, which causes a failure bumping the lockfile:

```
error: Package 'crypto-policies' has (currently) unsupported <lua> script in '%post'
```

The previous crypto-policies is pinned for now.

xref https://github.com/coreos/rpm-ostree/issues/749

**Comment by bgilbert** [comment]
This is blocking https://github.com/coreos/fedora-coreos-config/pull/480.

**Comment by bgilbert** [comment]
The Fedora package maintainer [CLOSED WONTFIX](https://bugzilla.redhat.com/show_bug.cgi?id=1847454).

**Comment by jlebon** [comment]
We discussed this in the community meeting today:

```
#agreed we will work around the crypto-policies issue for now in rpm-ostree. meanwhile, we will also pursue a path forward with upstream rpm to support passing lua scriptlets at runtime
```

Re.

> we will work around the crypto-policies issue for now in rpm-ostree

this is done in https://github.com/coreos/rpm-ostree/pull/2136. Will probably spin a new release this week to unblock this.

Re.

> meanwhile, we will also pursue a path forward with upstream rpm to support passing lua scriptlets at runtime

I filed https://github.com/rpm-software-management/rpm/issues/1273.

**Comment by jlebon** [comment]
https://github.com/coreos/coreos-assembler/pull/1551

**Comment by dustymabe** [comment]
I confirmed that crypto-policies in the latest `testing` release is `crypto-policies-20200610-1.git7f9d474.fc32.noarch`. 

The fix for this went into testing stream release `32.20200629.2.0`.
